src/app/core/component.ts

```diff
@@ -341,8 +341,8 @@ export default class Component {
                 bounds = render.bounds;
             }
         }
-
-        return bounds;
+        if (bounds != null) return bounds;
+        else throw new Error("Cannot have bounds of null");
```

Comment:
This error message should give some identification info of the component :
1. Component ID
2. Component type

This way people can debug easily whenever an error is thrown.